cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -205,45 +217,47 @@ public String toString() {
             for (boolean metric: metrics) {
                 for (Map kafkaConfig : kafkaConfigs) {
                     for (Map zooConfig : zooConfigs) {
-                        for (Storage storage : storageConfigs) {
-                            for (TopicOperatorSpec toConfig : toConfigs) {
-                                for (EntityOperatorSpec eoConfig : eoConfigs) {
-                                    KafkaListeners listeners;
-                                    if (shift)   {
-                                        listeners = new KafkaListenersBuilder()
-                                                .withNewPlain()
+                        for (Storage kafkaStorage : kafkaStorageConfigs) {
+                            for (SingleVolumeStorage zkStorage : zkStorageConfigs) {
+                                for (TopicOperatorSpec toConfig : toConfigs) {
+                                    for (EntityOperatorSpec eoConfig : eoConfigs) {
```

Comment:
I assume this means this parameterized test now executes more cases: Does it take a lot longer to execute as a result and do we think the extra coverage is worth the increased execution time?

Reply:
It's true but I didn't notice a huge increment (at least locally).

Reply:
We could revert back to the previous tests moving Zookeeper from using a `SingleVolumeStorage` to a `Storage` (but then we have to check at CO level that `jbod` type is not specified)